addons/mail/static/tests/discuss/message_pin/pinned_messages_tests.js

```diff
@@ -138,11 +134,12 @@ QUnit.test("Jump to message from notification", async (assert) => {
         });
     }
     const { openDiscuss } = await start();
-    await openDiscuss(channelId);
-    await click(".o-mail-Message:eq(0) [title='Expand']");
-    await click(".dropdown-item:contains(Pin)");
-    await click(".modal-footer button:contains(pin it)");
-    await click(".o_mail_notification a:contains(message)");
-    await nextTick();
-    assert.isVisible($(".o-mail-Message:contains(Hello world!)"));
+    openDiscuss(channelId);
+    await clickContains(":nth-child(1 of .o-mail-Message) [title='Expand']");
+    await clickContains(".dropdown-item", { text: "Pin" });
+    await clickContains(".modal-footer button", { text: "Yeah, pin it!" });
+    await contains(".o_mail_notification");
+    await contains(".o-mail-Thread", { scroll: "bottom" });
+    await clickContains(".o_mail_notification a", { text: "message" });
```

Comment:
Again not sure what it functionally means `.o_mail_notification a` with text `message`.
I could probably see if I run this test in `debug`, but not understanding by just reading the code of test shows some lack of info from this assertion